code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -83,6 +83,9 @@
 		move_to_delay - 2 = speed. */
 	UpdateSpeed()
 	. = ..()
+	if(SSmaptype.maptype in SSmaptype.citymaps)
+		if(vision_range > 7)
+			vision_range = 8
```

Comment:
```suggestion
		vision_range = min(vision_range, 8)
```

Reply:
Tested and seems to work just as well. Ill do a quick second test on a non city map to make sure it doesnt effect normal LC.

Reply:
Okay works flawlessly.

Reply:
well, because its basically the same
min() takes in some arguments, and puts out whatever's the lowest. Thus if the vision_range is higher than 8 its set to it